src/components/SearchItem/SearchItem.tsx

```diff
@@ -23,20 +30,22 @@ function SearchItem({ onClick, ...props }: SearchItemProps) {
         }}
       ></input>
       <div className="flex gap-2.5" onClick={onClick}>
-        <img
+        <Image
           src={props.image}
+          width={105}
+          height={150}
           alt={props.fullName}
```

Comment:
```suggestion
        <Image
          src={props.image}
          width={105}
          height={150}
          priority
          alt={props.fullName}
```

I noticed a console warning that this image was detected as the Largest Contentful Paint (LCP). This means it's the most significant element on the page visible to the user.

To improve page load performance and user experience, please add the `priority` property to this `Image` component. This will instruct the browser to preload the image with high priority.

https://nextjs.org/docs/api-reference/next/image#priority